### PR TITLE
VideoPlayer: add audiosetting for downmix clev

### PR DESCRIPTION
**Pull request by FernetMenta**
see title

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -21489,3 +21489,13 @@ msgstr ""
 msgctxt "#39111"
 msgid "original"
 msgstr ""
+
+#: xbmc/video/dialogs/GUIDialogAudioSettings.cpp
+msgctxt "#39112"
+msgid "Downmix: Center Mix Level"
+msgstr ""
+
+#: xbmc/video/dialogs/GUIDialogAudioSettings.cpp
```
[the remark this answers is not in the document]
Comment (FernetMenta) [reply addons/resource.language.en_gb/resources/strings.po]: any suggestion? imo this is self explanatory and does not need further explanation provided s.o. is familiar with downmixing. this is not the right place to explain technology.

[the remark this answers is not in the document]
Comment (ksooo) [reply addons/resource.language.en_gb/resources/strings.po]: > provided s.o. is familiar with downmixing. this is not the right place to explain technology.

We cannot assume that a translator is familar with all technology used in Kodi. That's exactly why some explanatory words are of big value here. I think you don't need to explain the whole technology, but if you want high quality translations, some guiding words do not harm.

File: addons/resource.language.en_gb/resources/strings.po [hunk 2]
```diff
@@ -21489,3 +21489,13 @@ msgstr ""
...
+msgctxt "#39113"
+msgid "Center Mix Level in DB relative to metadata or default (-3 dB)"
```
Comment (ksooo) [inline addons/resource.language.en_gb/resources/strings.po]: "DB" => "dB"

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (notspiff) [reply addons/resource.language.en_gb/resources/strings.po]: my suggestion for a comment would be 'Used for a setting which allows to increase the weight given to the center channel when downmixing. Typically used to boost voice levels.'

[the remark this answers is not in the document]
Comment (FernetMenta) [reply addons/resource.language.en_gb/resources/strings.po]: thanks, added

**Comment by chewitt** [comment]
@FernetMenta please add this context description: "Used for a setting which allows centre channel (voice/speech) volume to be boosted when downmixing"

**Comment by zag2me** [comment]
Thank you! Tested and working as expected on Android Shield. Improvement for the majority of Kodi users without configurable 5.1 setups, now we don't need to watch movies on a raspberry pi or SPMC builds 👍

+25db seems about right for me on my soundbar. For anyone needing to find this: Its in OSD >> Settings >> Audio Settings >> "Downmix: Center Mix Level".

**Comment by fritsch** [comment]
I don't think so. As 2.0 is output and we downmix. But obviously clear 25
is way too much in general.

Could be that the 25 goes wrongly computed into the system :-)

Rainer Hochecker <notifications@github.com> schrieb am Di., 21. Aug. 2018,
21:21:

**Comment by FernetMenta** [comment]
screen shot 2018-08-22 at 07 23 21

@fritsch the setting is computed correctly

**Comment by zag2me** [comment]
Reported a few times that this setting does not "stick".

https://forum.kodi.tv/showthread.php?tid=334664&pid=2761012#pid2761012

**Pull request merged**